server/models/Tier.js

```diff
@@ -69,6 +69,10 @@ export default function(Sequelize, DataTypes) {
         type: DataTypes.ARRAY(DataTypes.INTEGER),
       },
 
+      minimumAmount: {
+        type: DataTypes.INTEGER,
```

Comment:
I recently made changes to enforce more restrictive validations on models, like in https://github.com/opencollective/opencollective-api/pull/1776/files#diff-768438be827d00a869e92ca592010413R201

Here we know that min amount should never be less than `0`.

Reply:
Good feedback, implementing that myself. cc @flickz

Reply:
By the way @Betree, why is `min` in `amount` not in a `validate` object?

```
      amount: {
        type: DataTypes.INTEGER, // In cents
        min: 0,
      },
```

Reply:
@znarf Seems like a mistake from https://github.com/opencollective/opencollective-api/commit/616100eedd229548224810f599539846d81ce9e3. I missed it while adding validations, I'll fix it myself